Dockerfile

```diff
@@ -7,4 +15,6 @@ COPY . .
 # docker daemon maps app's port
 EXPOSE 8000
 
+RUN npm install
```

Comment:
Have the container install its dependencies instead of including `node_modules` in the container. (Not sure if this is a good pattern or an anti-pattern, tbh. Would like feedback from folks with more Docker experience.)

Reply:
I think we were caching node_modules to speed up our builds and to aid in deployment?  Maybe?  @jermnelson can confirm.